package.json

```diff
@@ -42,7 +42,7 @@
         "insertGuid.pasteAutomatically": {
           "type": "string",
           "default": "",
-          "description":"Paste GUID without selection, using the specified format. Available GUID formats are:\n\t{d}|{D}: Simple string in lowercase or uppercase\n\t{n}|{N}: Unformatted string in lowercase or uppercase\n\nOther characters will be interpreted literally.\nExamples:\n\t'{n}'\n\t\t=> '03afd966b18446079325cd58313ad637'\n\tnew GUID(\"{D}\")]\n\t\t => new GUID(\"123CF48A-16ED-469D-B38E-A465DDCD667E\")]\n"
+          "markdownDescription":"Paste GUID without selection, using the specified format. Available GUID formats are:  \n&nbsp;&nbsp;&nbsp;&nbsp;*{b}|{B}*: Simple string in lowercase or uppercase  \n&nbsp;&nbsp;&nbsp;&nbsp;*{n}|{N}*: Unformatted string in lowercase or uppercase  \n&nbsp;  \nOther characters will be interpreted literally.  \n&nbsp;  \nExamples:  \n&nbsp;&nbsp;&nbsp;&nbsp;*'{n}'* => '03afd966b18446079325cd58313ad637'  \n&nbsp;&nbsp;&nbsp;&nbsp;*new GUID(\"{B}\")* => new GUID(\"123CF48A-16ED-469D-B38E-A465DDCD667E\")"
```

Comment:
Since it's markdown, use "\n* <text>" for bullets, which corresponds to:

```
* {b}|{B}
* {d}|{D}
* ...
```

If you're passing markdown, don't render bullets yourself. It will do that for you.

Reply:
I didn't render bullets; I wanted just indentation. The surrounding *'s were markdown for 'italics'. I updated to bullets per your comment though

![image](https://user-images.githubusercontent.com/1996525/122658953-ac962300-d1ad-11eb-8bfa-c4d6d781a2c4.png)